DXMainClient/DXGUI/Multiplayer/GameLobby/GameLobbyBase.cs

```diff
@@ -299,133 +239,64 @@ public override void Initialize()
+                loadOrSaveGameOptionPresetWindow.PresetLoaded += (sender, s) => HandleGameOptionPresetLoadCommand(s);
+                loadOrSaveGameOptionPresetWindow.PresetSaved += (sender, s) => HandleGameOptionPresetSaveCommand(s);
+                loadOrSaveGameOptionPresetWindow.Disable();
+                var loadConfigMenuItem = new XNAContextMenuItem()
+                {
+                    Text = "Load",
+                    SelectAction = () => loadOrSaveGameOptionPresetWindow.Show(true)
+                };
+                var saveConfigMenuItem = new XNAContextMenuItem()
+                {
+                    Text = "Save",
+                    SelectAction = () => loadOrSaveGameOptionPresetWindow.Show(false)
+                };
 
-            AddChild(PlayerOptionsPanel);
-            AddChild(btnLaunchGame);
-            AddChild(btnLeaveGame);
-            AddChild(btnPickRandomMap);
-        }
+                loadSaveGameOptionsMenu = new XNAContextMenu(WindowManager);
+                loadSaveGameOptionsMenu.Name = nameof(loadSaveGameOptionsMenu);
+                loadSaveGameOptionsMenu.ClientRectangle = new Rectangle(0, 0, 75, 0);
+                loadSaveGameOptionsMenu.Items.Add(loadConfigMenuItem);
+                loadSaveGameOptionsMenu.Items.Add(saveConfigMenuItem);
 
-        private void InitializeGameOptionsPanel()
-        {
-            GameOptionsPanel = new XNAPanel(WindowManager);
-            GameOptionsPanel.Name = nameof(GameOptionsPanel);
-            GameOptionsPanel.ClientRectangle = new Rectangle(Width - 411, 12, 399, 289);
-            GameOptionsPanel.BackgroundTexture = AssetLoader.CreateTexture(new Color(0, 0, 0, 192), 1, 1);
-            GameOptionsPanel.PanelBackgroundDrawMode = PanelBackgroundImageDrawMode.STRETCHED;
-
-            loadOrSaveGameOptionPresetWindow = new LoadOrSaveGameOptionPresetWindow(WindowManager);
-            loadOrSaveGameOptionPresetWindow.Name = nameof(loadOrSaveGameOptionPresetWindow);
-            loadOrSaveGameOptionPresetWindow.PresetLoaded += (sender, s) => HandleGameOptionPresetLoadCommand(s);
-            loadOrSaveGameOptionPresetWindow.PresetSaved += (sender, s) => HandleGameOptionPresetSaveCommand(s);
-            loadOrSaveGameOptionPresetWindow.Disable();
-            var loadConfigMenuItem = new XNAContextMenuItem()
-            {
-                Text = "Load",
-                SelectAction = () => loadOrSaveGameOptionPresetWindow.Show(true)
-            };
-            var saveConfigMenuItem = new XNAContextMenuItem()
-            {
-                Text = "Save",
-                SelectAction = () => loadOrSaveGameOptionPresetWindow.Show(false)
-            };
+                BtnSaveLoadGameOptions.LeftClick += (sender, args) =>
+                {
+                    loadSaveGameOptionsMenu.Open(new Point(BtnSaveLoadGameOptions.X - 74, BtnSaveLoadGameOptions.Y));
+                };
```

Comment:
Not sure if the braces are needed here.

Reply:
Although the brackets around the lambda expression are indeed optional, the meaning is different.
So be careful if you want to remove the brackets.
e.g. The following two statements are equals.
`x => { return x + 1; }`
`x => x + 1`